### DISCUSSION TITLE
Signac for Macaque - Annotation issues

### DISCUSSION THREAD

**Discussion by khanthanan** [stuart-lab/signac#1188] [Q&A] [answered]
Hi, 
Thank you very much for this awesome platform. I'm currently working on a multiome dataset for Macaque. I ran across some issues with the annotations. I started by downloading the gtf file from ensemble website. 

`
gtffile <- "Macaca_mulatta.Mmul_10.107.gtf.gz"
`
`DB <- ensDbFromGtf(gtf = gtffile)
`
`EDB <- EnsDb(DB)
`
`annotation <- GetGRangesFromEnsDb(ensdb = EDB)
`
When I checked whether the annotations and fragments have the same chrom name, i found that annotations didn't have a column for chrom. I saw that many people had this issue (for human data), and the solution was to update the package.
 
![image](https://user-images.githubusercontent.com/89409347/184642799-59832349-2e52-44fd-85cd-30d5d2fdb8b2.png)

 
I'd highly appreciate it if you could help me with this. 

Thank you. 

![image](https://user-images.githubusercontent.com/89409347/184642623-01c6560f-d251-4897-bc57-d92318098830.png)

**Comment by timoast** [comment, marked as answer]
The gene annotations are in GenomicRanges format, which uses "seqnames" as the chromosome names. Here it looks like you need to prefix the seqnames in the annotation with "chr" to make them match the fragment file.

**Comment by khanthanan** [comment]
Thank you very much for your quick response. I just made the changes and it finally worked. I really appreciate your help. 

Best

**Reply by wxpbioinfo** [reply]
> Thank you very much for your quick response. I just made the changes and it finally worked. I really appreciate your help.
> 
> Best

Hello,I met the same problem, but I indicated that NA existed here. Could you please tell me how to solve it? Thank you!
![image](https://user-images.githubusercontent.com/115339890/220287293-73a535a2-a701-4a42-849d-33a620106129.png)
